2.0.0/Dockerfile

```diff
@@ -26,6 +26,7 @@ ENV WEBSITE_ROLE_INSTANCE_ID localRoleInstance
 ENV WEBSITE_INSTANCE_ID localInstance
 ENV PATH ${PATH}:/home/site/wwwroot
 ENV ASPNETCORE_URLS=
+ENV ASPNETCORE_FORWARDEDHEADERS_ENABLED=true
```

Comment:
Have we used `true` or `1` for other variables like this, @Tratcher?

Reply:
We don't usually set env vars, but when we read them we usually accept `true/false` or `1/0`.

Reply:
Cool, `true` sounds good then.